src/com/esotericsoftware/kryo/util/ObjectMap.java

```diff
@@ -45,8 +45,8 @@
 
 	public int size;
 
-	K[] keyTable;
-	V[] valueTable;
+	protected K[] keyTable;

```

Comment:
We can keep these fields package private, since the new map is in the same package.